documentation/source/contributing/licenses.rst

```diff
@@ -0,0 +1,154 @@
+Licenses
+========
+
+In general we accept every wide acknowledged license, which allows to share, modify, share the modified work, make commercial use or any combination of these. In following you see a list of licenses we accept and below a list of licenses we don't, if you can't find a license in these lists and still unsure, don't hesitate to ask us about it on our `Discord <https://discord.com/invite/acUW8k7>`__.
+
+List of accepted licenses
```

Comment:
Would make it a list we do not need a table which describes the specifications of the licenses, that should not be the topic here.
So basically a list which licenses we accept and which we do not, and a hint that not listed once must have some points we expect like commercial usage.

Reply:
Done.